src/api_client.rs

```diff
@@ -5,16 +5,18 @@ use crate::error::ApiError;
 use crate::types::{
     ApiResult, AttestationDuty, BalanceSummary, BeaconHeaderSummary, BeaconProposerRegistration,
     BlockId, CommitteeDescriptor, CommitteeFilter, CommitteeSummary, EventTopic,
-    FinalityCheckpoints, GenesisDetails, HealthStatus, NetworkIdentity, PeerDescription,
-    PeerDescriptor, PeerSummary, ProposerDuty, PublicKeyOrIndex, RootData, StateId,
-    SyncCommitteeDescriptor, SyncCommitteeDuty, SyncCommitteeSummary, SyncStatus, ValidatorStatus,
-    ValidatorSummary, Value, VersionData,
+    FinalityCheckpoints, GenesisDetails, HealthStatus, PeerSummary, ProposerDuty, PublicKeyOrIndex,
+    RootData, StateId, SyncCommitteeDescriptor, SyncCommitteeDuty, SyncCommitteeSummary,
+    SyncStatus, ValidatorStatus, ValidatorSummary, Value, VersionData,
 };
+#[cfg(feature = "peer-id")]
+use crate::types::{NetworkIdentity, PeerDescription, PeerDescriptor};
```

Comment:
```suggestion
use crate::types::{NetworkIdentity, PeerDescription};
use crate::types::PeerDescriptor;
```

and this last item gets merged into the decl above w/ the rest of the types